### PR TITLE
Use image-tag from build-tools

### PR DESCRIPTION
**Pull request by ekimekim**
Update `./tools` to latest build-tools master
and replace `image-tag` script with the newly-shared version

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -69,12 +69,12 @@ deployment:
           docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS &&
           (test "${DOCKER_ORGANIZATION:-$DOCKER_USER}" == "weaveworks" || (
             docker tag weaveworks/scope:latest ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:latest &&
-            docker tag weaveworks/scope:$(./image-tag) ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:$(./image-tag)
+            docker tag weaveworks/scope:$(./tools/image-tag) ${DOCKER_ORGANIZATION:-$DOCKER_USER}/scope:$(./image-tag)
```
Comment (tomwilkie) [inline circle.yml]: Missed a reference to ./image-tag?

**Comment by tomwilkie** [comment]
@ekimekim ping?

File: circle.yml (hunk 1 above)
Comment (ekimekim) [inline circle.yml]: oops, good catch.

**Comment by ekimekim** [comment]
fixed, ptal

**Comment by tomwilkie** [comment]
LGTM

**Pull request merged**